registry.json

```diff
@@ -4425,5 +4425,20 @@
    "telegram": "lunarpool_io",
    "operatorName": "Dusan",
    "github": "lunarpool"
+  },
+  {
+   "ticker": "BLU",
+   "poolId": "b9147445047312aa139a331390bba715eebb58f1b66c96e660e48986",
+   "social": {
+     "website": "https://www.adablue.io/",
+     "twitter": "ada_blu_",
+     "discord": "",
+     "facebook": "",
+     "instagram": "",
+     "youtube": ""
+   },
+   "telegram": "",
```

Comment:
As per alliance requirements you must have a telegram id. This must be for an individual and not a channel or group